Client/src/Pages/Status/CreateStatus/index.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from "prop-types";
+import { useNavigate } from "react-router";
+import { useEffect, useState } from "react";
+import { Box, Tab, useTheme } from "@mui/material";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import GeneralSettingsPanel from "../../../Components/TabPanels/Status/GeneralSettingsPanel";
+import {
+	capitalizeFirstLetter,
+	toLowerCaseFirstLetter,
+} from "../../../Utils/stringUtils";
+import ContentPanel from "../../../Components/TabPanels/Status/ContentPanel";
+
+/**
+ * CreateStatus page renders a page with tabs for general settings and contents.
+ * @param {string} [props.open] - Specifies the initially open tab: 'general settings' or 'content'.
+ * @returns {JSX.Element}
+ */
+
+const CreateStatus = ({ open = "general-settings" }) => {
+	const theme = useTheme();
+	const navigate = useNavigate();
+	let tabList = ["General Settings", "Contents"];
+	const tab = open
+		.split("-")
+		.map((a) => capitalizeFirstLetter(a))
+		.join(" ");
+
+	const [tabIdx, setTabIdx] = useState(tabList.indexOf(tab));
```

Comment:
If we are changing tab navigation through the url we probably won't need the state

Reply:
I found the tabIdx state is in fact needed for actually showing the respective tab panel

Reply:
Not if we get our state from the url. Call me if you want me to clarify that